<launch>
  <node pkg="roboteq_bridge" type="drive_node" name="roboteq">
		<param name="port" value="/dev/ttyACM1"/>
    <param name="baud" value="115200"/>
    <param name="watchdog_timeout" value="100"/> <!-- millisec -->
    <param name="encoder_ppr" value="2048"/>
    <param name="gear_ratio" value="18.7"/>
    <param name="wheel_radius" value="0.167"/> <!-- meter -->
    <param name="wheel_separation" value="0.56"/> <!-- meter -->
    <!-- report angular velocities or tangent velocities -->
    <param name="report_angular_velocities" value="true"/>
    <param name="motor_amp_limit" value="50"/> <!-- Amper -->
    <param name="motor_max_speed" value="100"/> <!-- rpm -->
    <param name="motor_max_acceleration" value="200"/> <!-- rpm/sec -->
    <param name="motor_max_deceleration" value="200"/> <!-- rpm/sec -->
	</node>
</launch>
